docs/application/native/guides/multimedia/image-edit.md

```diff
@@ -234,36 +335,74 @@ To crop an image:
 1. Create a transformation handle using the `image_util_transform_create()` function:
 
    ```
-   transformation_h handle;
-   ret = image_util_transform_create(&handle);
+   transformation_h transform_h;
+   ret = image_util_transform_create(&transform_h);
    ```
 
 2. Set the crop area using the `image_util_transform_set_crop_area()` function:
 
    ```
-   ret = image_util_transform_set_crop_area(handle, start_x, start_y, end_x, end_y);
+   ret = image_util_transform_set_crop_area(transform_h, start_x, start_y, end_x, end_y);
    ```
 
 3. Execute the transformation using the `image_util_transform_run()` function:
 
    ```
-   ret = image_util_transform_run(handle, src,
-                                  (image_util_transform_completed_cb)completed_callback,
-                                  user_data);
+   ret = image_util_transform_run2_async(transform_h, src_image,
+                                        (image_util_transform_completed2_cb)completed_callback,
+                                        user_data);
    ```
 
    > **Note**
    >
    > - Because of a YUV restriction, and because the crop start position can be set arbitrarily, the cropped image width and height must be even.
-   > - If hardware acceleration is enabled, you can execute 2 more image transformations using the same transformation handle.
+
+4. Handle the transformation results in the `image_util_transform_completed2_cb()` callback, which is invoked after the transformation is complete.
+
+5. After the transformation is complete, destroy the transformation handle using the `image_util_transform_destroy()` function:
+
+   ```
+   ret = image_util_transform_destroy(transform_h);
+   ```
+
+## Cropping a Media Packet
+
+To crop a media_packet:
+
+1. Create a transformation handle using the `image_util_transform_create()` function:
+
+   ```
+   transformation_h transform_h;
+   ret = image_util_transform_create(&transform_h);
+   ```
+
+2. Set the crop area using the `image_util_transform_set_crop_area()` function:
+
+   ```
+   ret = image_util_transform_set_crop_area(transform_h, start_x, start_y, end_x, end_y);
+   ```
+
+3. Execute the transformation using the `image_util_transform_run()` function:
+
+   ```
+   ret = image_util_transform_run(transform_h, (media_packet_h)src_packet,
+                                (image_util_transform_completed_cb)completed_callback,
+                                user_data);
+   ```
+
+   > **Note**
```

Comment:
@surbhig12 
"If the color space is YUV, then the width and height of the target image must be multiples of eight."
Why did you add "then"? Did you want to emphasis the dependency between main clause and subordinate clause.

We usually write "if..., ..." without "then" except mandatory case.

You had better NOT change when there is no problem in a sentence.

Reply:
@journey2w,
The intention to use "then" in this particular case was to stress and emphasize on the dependency. 
Yes, I agree that we usually write "if..., ..." without "then" except mandatory case.
However, this is a conditional clause and we use “then” if we need to stress. Since it was an important information, I used "then".
I referred to: https://english.stackexchange. com/questions/247323/dropping-then-in-if-statements.

Although, if you feel "then" is not required here. I would go with your suggestion.
Thank you for the explanation.